content/get-started/multisig-accounts.mdx

```diff
@@ -0,0 +1,201 @@
+---
+title: Multisig Accounts
+seo-title: Multisig Accounts
+---
+
+## Create a multisig account
+
+Iron Fish supports multisig accounts using [FROST](https://frost.zfnd.org/frost.html), a threshold signature scheme.
+
+### Create a participant identity
+
+Each signer or participant in an Iron Fish multisig account must first generate an identity
+
+```sh
+ironfish wallet:multisig:participant:create
+```
+
+The participant identity uniquely identifies the participant in the group. The identity is also a public key and allows other members of the group to encrypt data for that participant. The participant's secret key is stored in their Iron Fish wallet.
+
+### Generate multisig account imports
```

Comment:
There needs to be in this section a big warning to inform users about the expectations about the "trusted dealer". Something like this (can be improved):

```
>! **Note:** This process involves generating an authorizing key for the multisig account, and then splitting the key in multiple parts, one for each participants. This means that the Iron Fish node where `wallet:multisig:dealer:create` is run will hold in memory for a brief period of time the full (unsplit) authorizing key, which allows to authorize transactions. For this reason it is important that the dealer node is fully trusted, and that it runs in a secure compute environment. If an eavesdropper is able to get access to the internal memory of the machine where the dealer node runs, they may be able to get access to the full (unsplit) authorizing key.
```